<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorrowBookWindow</class>
 <widget class="QMainWindow" name="BorrowBookWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>3ds</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>借书</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_bookid">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_bookname">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_userid">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdb_bookid">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>书号</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdb_bookname">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>书名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>借书人ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_borrow">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>借书</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
